### DISCUSSION TITLE
How do you get the starstruck Badge?

### DISCUSSION THREAD

**Discussion by mlfanatic2024** [drknzz/GitHub-Achievements#394] [Q&A] [answered]
How do you get the starstruck Badge?

**Comment by Coolbreeze151** [comment, marked as answer]
To get the starstruck badge you basically need to get a repo with minimum of 16 stars. Having said that, it would be great if you can support me in getting more stars for my repo too! 

https://github.com/Coolbreeze151/BackendRoadmap

**Reply by felizardo27** [reply]
please helpe me, this is my https://github.com/felizardo27/pokedex-RN
